src/components/media-player/media-volume-slider.js

```diff
@@ -10,12 +10,21 @@ export const MEDIA_VOLUME_SLIDER_ORIENTATION = {
 };
 
 function MediaVolumeSlider({ value, onChange, orientation }) {
+  const [lastValueBeforeMutting, setLastValueBeforeMutting] = useState(value);
```

Comment:
Typo: `Muting`

Reply:
I appreciate you calling it a typo, given that I intentionally wrote it like that thinking it is correct 😶 
Corrected, thanks!

Reply:
:smile: